test-data/unit/check-namespaces.test

```diff
@@ -0,0 +1,77 @@
+-- Type checker test cases dealing with namespaces imports
```

Comment:
Might be good to also have a test for a "namespace" inside a package with `__init__.py` (e.g. `a/b/c.py` where `a/__init__.py` exists but `a/b/__init__.py` does not.) This isn't a _useful_ deployment of a namespace package, but it works fine in Py3 and thus is likely to occur in the wild by accident of omission.